src/Console/ConfigurationResolver.php

```diff
@@ -305,6 +305,10 @@ public function getDirectory()
     {
         if (null === $this->directory) {
             $path = $this->getCacheFile();
+            if (null === $path) {
+                return null;
```

Comment:
this explicitly violate the method return type

Reply:
Indeed... Modified, thanks :)